packages/react-ui/src/widgets/utils/formatterUtils.js

```diff
@@ -0,0 +1,5 @@
+export function processFormatterRes(formatterRes) {
```

Comment:
I would love to describe input using TS but we are only providing d.ts. In this case, I guess that the name is correct because we are processing the formatters response that we usually cover in other widgets. I find also ok: formatFomatterRes, but I don't see the problem with process word.

Reply:
Ah yes, sorry for the TS suggestion, doesn't apply here.

> I don't see the problem with process word.

Just a matter of personal taste probably, "process" kinda sounds to me like a full-blown machinery will run. Also, it's a big vague, like, process to do what? But "formatFormatterRes" isn't great either, and as I said, it's a non-blocker for me, let's move on with that :slightly_smiling_face: